apps/namadillo/src/atoms/integrations/functions.ts

```diff
@@ -370,3 +370,37 @@ export const createIbcTx = async (
   );
   return transactionPair;
 };
+
+export const namadaLocal = (chainId: string): Chain => ({
+  ...internalDevnetChain,
+  chain_name: "localnet",
+  chain_id: chainId,
+});
+
+export const namadaLocalAsset = (tokenAddress: string): AssetList => ({
+  ...internalDevnetAssets,
+  chain_name: "localnet",
+  assets: internalDevnetAssets.assets.map((asset) =>
+    asset.name === "NAM" ?
```

Comment:
I think this is not going to work once we're using the registry with anoma/namada-chain-registry#13 merged. Maybe could use `"Namada"` or check against `symbol` or `base` instead?